M2/Macaulay2/packages/Points.m2

```diff
@@ -737,16 +737,16 @@ doc ///
     i = randomPoints(r,n)
    Inputs
     r:ZZ
-     number of points
-    n:ZZ
      ambient dimension
+    n:ZZ
+     number of points
    Outputs
     i:Ideal
      ideal of the random points
    Description
     Text
      The script defines a ring R with r+1 variables, and calls
-     points(R,randomPointsMat(r,n))
+     points(R,randomPointsMat(R, n))
```

Comment:
Should the second `R` be `r`?

Reply:
Definitely `R`,  the first input to `randomPointsMat` is a ring:
https://github.com/Macaulay2/M2/blob/73e7016b27dd79d4c115b4821f5a6ab05ca94212/M2/Macaulay2/packages/Points.m2#L582-L587